extensions/grpc/deployment/src/main/java/io/quarkus/grpc/deployment/GrpcBuildTimeConfig.java

```diff
@@ -12,4 +12,11 @@
     @ConfigItem(name = "metrics.enabled")
     public boolean metricsEnabled;
 
+    /**
+     * Start gRPC server in dev mode even if no gRPC services are implemented.
+     * By default set to `true` to ease incremental development of new services using dev mode.
+     */
+    @ConfigItem(name = "dev-force-server-start", defaultValue = "true")
```

Comment:
I think that it might make sense to introduce a `@ConfigGroup` for the dev mode, e.g. `@ConfigItem
    public GrpcDevModeConfig devMode;` with `@ConfigItem(name = "force-server-start", defaultValue = "true") public boolean forceServerStart;` so that the final name of the property would be something like `quarkus.grpc.dev-mode.force-server-start`. WDYT?

Reply:
done